.github/workflows/provider-publish-service-artifacts.yml

```diff
@@ -42,7 +42,7 @@ env:
   
 jobs:
   index:
-    runs-on: [e2-standard-8, linux]
```

Comment:
To be reverted back before being merged. Had to replace the runners to be able to test.